### DISCUSSION TITLE
Dependency XXXX has unsupported value main

### DISCUSSION THREAD

**Discussion by faradaytrs** [renovatebot/renovate#20267] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34

### If you're self-hosting Renovate, select which platform you are using.

github.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate to update submodules

### What you tried so far.

I am trying to use self-hosted renovate for private repo with private submodules. I know it's not officially supported, but i believe I found some advices here and got custom ssh layer with auth.

Cloning submodules is successful, but later I see the following message in the logs:

````
Dependency XXXXX has unsupported value main
````

main is the branch of submodule. I have several submodules, all of them have this problem.
As result, nothing inside of submodule is updated by renovate.

Here is my config for monorepo:
````json
{
  "extends": ["config:base"],
  "rebaseStalePrs": true,
  "ignoreTests": false,
  "automerge": true,
  "automergeType": "branch",
  "packageRules": [
    {
      "depTypeList": ["dependencies"],
      "updateTypes": ["major"],
      "automerge": false
    }
  ],
  "git-submodules": {
    "enabled": true
  },
  "hostRules": [
    {
      "matchHost": "github.com",
      "token": "XXX_TOKEN___XXXX"
    }
  ]
}
````

### Relevant debug logs

Here are my logs:
[output.log](https://github.com/renovatebot/renovate/files/10672555/output.log)

**Comment by rarkins** [comment]
`Dependency XXXXX has unsupported value main` is not a problem. It's a debug log, not an error
